CMakeLists.txt

```diff
@@ -1,5 +1,15 @@
 cmake_minimum_required(VERSION 3.20)  # For using CMAKE_<LANG>_BYTE_ORDER
 
+if (NOT DEFINED ENV{VCPKG_ROOT})
+  message(FATAL_ERROR "
+    Please set an environment variable VCPKG_ROOT
+    For example:
+    export VCPKG_ROOT=/path/to/vcpkg
+    ")
+endif()
```

Comment:
interesting thought. in some sense, yes - i think you would not be able to _build_ using a system-provided library.

HOWEVER, as i understand it, SDL and zlib are both dynamically linked - correct? if that's the case, then you could still _run_ Descent3 using the system-provided SDL and zlib, by having the binary load up the system libs instead of the vcpkg ones. all that is really needed for the _build_ are the headers and symbol stubs, which I do think we should keep strictly specified.

Reply:
I was incorrect here: We are currently linking SDL statically.

Reply:
This adds additional dependency vcpkg for users on Linux and macOS, which on some cases undesirable (Linux distribution enforces using external dependencies from thier repositories for security reasons). We should keep ability to install external dependencies without using vcpkg.